### PR TITLE
Introduce a Dockerfile using Fedora 25 nightly builds of FreeIPA master

### PR DESCRIPTION
**Pull request by martbab**
This Dockerfile is required for the early testing of the influence of
the current devel snapshots of FreeIPA on the stability of the container
image.

The images build from this Dockerfile should not be pushed to registry.
Since it contains development snapshots you need to rebuild it often
anyway.

### CODE REVIEW

File: Dockerfile.fedora-25-master-nightly [hunk 1]
```diff
@@ -0,0 +1,87 @@
+# Clone from the Fedora 25 image
+FROM fedora:25
+
+MAINTAINER IPA Maintainers <ipa-maint@redhat.com>
```
Comment (adelton) [inline Dockerfile.fedora-25-master-nightly]: s/IPA/FreeIPA/

Do you really want to advertise this email address?

File: Dockerfile.fedora-25-master-nightly [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+RUN mkdir -p /run/lock \
+    && cd /etc/yum.repos.d \
+    && curl -O https://copr.fedorainfracloud.org/coprs/g/freeipa/freeipa-master/repo/fedora-25/group_freeipa-freeipa-master-fedora-25.repo \
+    && curl -O https://copr.fedorainfracloud.org/coprs/g/freeipa/freeipa-master-nightly/repo/fedora-25/group_freeipa-freeipa-master-nightly-fedora-25.repo \
+    && dnf install -y freeipa-server \
+    freeipa-server-dns \
+    freeipa-server-trust-ad \
+    initscripts \
+    --best --allowerasing \
```
Comment (adelton) [inline Dockerfile.fedora-25-master-nightly]: Is the `--allowerasing` actually needed?

Comment (martbab) [inline Dockerfile.fedora-25-master-nightly]: I would rather keep it there to take care of incidental dependency conflicts that happen sometimes when using nightly builds.

**Comment by martbab** [comment]
I have used the address of upstream devel list in the MAINTAINER label.

**Comment by adelton** [comment]
LGTM.

**Pull request merged**